storage/innobase/xtrabackup/test/t/undo_tablespaces.sh

```diff
@@ -36,9 +36,7 @@ start_server
 
 # create some undo tablespaces
 mysql -e "CREATE UNDO TABLESPACE undo1 ADD DATAFILE '$undo_directory_ext/undo1.ibu'"
-mysql -e "CREATE UNDO TABLESPACE undo2 ADD DATAFILE '$undo_directory_ext/undo2.ibu'"
-mysql -e "CREATE UNDO TABLESPACE undo3 ADD DATAFILE 'undo1.ibu'"
-mysql -e "CREATE UNDO TABLESPACE undo4 ADD DATAFILE 'undo2.ibu'"
```

Comment:
Please keep this scenario as upstream allows it (until it gets fixed) and show that backup detects the conflict and aborts the backup